app/views/people/_announcements.html.erb

```diff
@@ -1,5 +1,5 @@
 <% if person.has_announcements? %>
-  <section id="announcements">
+  <section id="announcements" dir="<%= page_text_direction %>" lang="<%= person.locale %>">
```

Comment:
Since this contains a left to right language (in this instance, 'Announcements') that isn't translated it shouldn't be displayed right-to-left.

It's also putting out lang regardless of the default language, which seems a bit messy - on frontend there's a helper that adds the lang attribute only when the language is different to the default language (https://github.com/alphagov/frontend/blob/master/app/helpers/locale_helper.rb) which would prevent attributes being added when they're not needed.